spec/models/spree/stock_location_spec.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+module Spree
+  describe StockLocation do
+    subject { create(:stock_location_with_items, backorderable_default: true) }
+    let(:stock_item) { subject.stock_items.order(:id).first }
+    let(:variant) { stock_item.variant }
+
+    it 'creates stock_items for all variants' do
+      expect(subject.stock_items.count).to eq Variant.count
+    end
+
+    context "handling stock items" do
+      let!(:variant) { create(:variant) }
+
+      context "given a variant" do
+        context "propagate all variants" do
+          subject { StockLocation.new(name: "testing") }
+
+          before { subject.propagate_all_variants = true }
```

Comment:
This no longer needed, right?

Reply:
ah, yes, thanks! removed in a new commit :+1: